.github/workflows/therock-ci-windows.yml

```
@@ -91,17 +88,18 @@ jobs:
         timeout-minutes: 30
         run: |
           git config --global core.longpaths true
-          python ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-rocm-systems --no-include-ml-frameworks
+          python ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-rocm-systems --no-include-rocm-libraries --no-include-ml-frameworks
+          dvc pull
 
       - name: Configure Projects
         env:
           amdgpu_families: ${{ env.AMDGPU_FAMILIES }}
           package_version: "ADHOCBUILD"
-          extra_cmake_options: "-DBUILD_TESTING=OFF -DTHEROCK_USE_EXTERNAL_ROCM_LIBRARIES=ON -DTHEROCK_ROCM_LIBRARIES_SOURCE_DIR=${{ github.workspace }} ${{ inputs.cmake_options }}"
+          extra_cmake_options: "-DTHEROCK_USE_EXTERNAL_ROCM_SYSTEMS=ON -DTHEROCK_ROCM_SYSTEMS_SOURCE_DIR=${{ github.workspace }} ${{ inputs.cmake_options }}"
```

```suggestion
          extra_cmake_options: "-DTHEROCK_ROCM_SYSTEMS_SOURCE_DIR=${{ github.workspace }} ${{ inputs.cmake_options }}"
```
Does `github.workspace` because this does not run in a container? Might want to use the same as in Linux here for consistency though.

---

`github.workspace` unfortunately doesn't work as there is an issue with the container/work paths. I also tried `$GITHUB_WORKSPACE`, but it never considered it to be set.

---

Yeah, I rather meant to use `../` on Windows if that is legit.

---

ah, i can update in future PR. a patch was landed so need to add a change there

---

Not too important but could be done somewhen to increase consistency as a drive by maybe next time :)